identifiers/country-uy/departments.csv

```diff
@@ -0,0 +1,20 @@
+id,name
+ocd-division/country:uy/department:uy-ar,Artigas

```

Comment:
I think we shuold drop the `uy-` prefex from the department identifiers since it's redundant with the `country:uy` portion.

Reply:
Sounds good, I just removed them.